microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -424,22 +443,23 @@ Whether to use local overrides for response headers.
 <!-- todo: need text -->
 
 Status:
-*  This checkbox is present in Microsoft Edge Stable v103.
-*  This checkbox is present in Microsoft Edge Canary v105.
+*  This checkbox is present in Microsoft Edge Canary v106.
+*  This checkbox is present in Microsoft Edge Stable v104.
 
 
 <!-- ====================================================================== -->
 ## Enable CSS Authoring hints for inactive rules, deprecated properties, etc.
 <!-- keep "etc." in heading, to match UI label -->
-<!-- new in v105 -->
+
+_new as of v105_
 
 Whether to enable CSS authoring hints in the **Styles** panel of the **Elements** tool to display information tooltips next to inactive or deprecated properties.
 
-In Canary v105, this checkbox has no effect.  When this feature is implemented, this feature will provide help when writing or inspecting CSS in the **Styles** panel.  Small information icons will appear next to CSS properties that are inactive (or deprecated).
+In Canary v105 and later, this checkbox has no effect.  When this feature is implemented, this feature will provide help when writing or inspecting CSS in the **Styles** panel.  Small information icons will appear next to CSS properties that are inactive (or deprecated).
```

Comment:
@captainbrosset - pls review/clarify desc

Reply:
```suggestion
This feature provides help when writing or inspecting CSS in the **Styles** panel.  Small information icons appear next to CSS properties that are inactive or deprecated. An inactive property is one that applies correctly to an element but has no effect on it (for example `width:100px` applied to an inline element has no effect).
```

Reply:
Removed the "has no effect" part because the feature does work with 105.